AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -25,6 +25,7 @@ public protocol CardComponentDelegate: AnyObject {
 /// A component that provides a form for card payments.
 public class CardComponent: PaymentComponent, PresentableComponent, Localizable, Observer, LoadingComponent {
 
+    private let publicBinLenght = 6
```

Comment:
length*

Reply:
👍

Reply:
PR for it [here](https://github.com/Adyen/adyen-ios/pull/448)